broccoli/packages.js

```diff
@@ -83,6 +84,12 @@ module.exports.getPackagesES = function getPackagesES() {
     exclude: ['**/*.ts'],
   });
 
+  // tsc / typescript handles decorators and class properties on its own
+  // so for non ts, transpile the proposal features (decorators, etc)
+  let transpiledProposals = transfromBabelPlugins(
+    debugTree(nonTypeScriptContents, `get-packages-es:babel-plugins`)
```

Comment:
I would suggest to put a `debugTree` inside **and** outside of it. `get-packages-es:babel:input` inside, and `get-packages-es:babel:output` outside

Reply:
done :)